core/src/main/java/hudson/slaves/OfflineCause.java

```diff
@@ -54,17 +55,19 @@
      */
     public static class SimpleOfflineCause extends OfflineCause {
         public final Localizable description;
+        public final Date timestamp;
```

Comment:
Thanks – it passed, and I have pushed the changes to this pull-request.

From: Oliver Gondža [mailto:notifications@github.com]
Sent: Monday, March 23, 2015 11:13 AM
To: jenkinsci/jenkins
Cc: Matthew Legrand (mmlegra)
Subject: Re: [jenkins] JENKINS-23197: add datestamp to node-offline message (#1605)


In core/src/main/java/hudson/slaves/OfflineCause.java<https://github.com/jenkinsci/jenkins/pull/1605#discussion_r26963271>:

> @@ -54,17 +55,19 @@

>       */

>      public static class SimpleOfflineCause extends OfflineCause {

>          public final Localizable description;

> +        public final Date timestamp;

mvn clean package -Dtest=NodeTest -DfailIfNoTests=false

—
Reply to this email directly or view it on GitHub<https://github.com/jenkinsci/jenkins/pull/1605/files#r26963271>.